packages/requisitions/src/RequestRequisition/DetailView/AppBarButtons/AppBarButtons.tsx

```diff
@@ -43,20 +41,9 @@ export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({
   return (
     <AppBarButtonsPortal>
       <Grid container gap={1}>
-        <ButtonWithIcon
-          disabled={isDisabled}
-          Icon={<PlusCircleIcon />}
-          label={t('button.add-from-master-list')}
-          onClick={modalController.toggleOn}
-        />
-        <MasterListSearchModal
-          open={modalController.isOn}
-          onClose={modalController.toggleOff}
-          onChange={masterList => {
-            modalController.toggleOff();
-            addFromMasterList(masterList);
-          }}
-        />
+        <UseSuggestedQuantityButton />
```

Comment:
could you change the order please?
I think: Add item should be first, since that's most often used, and should be easiest to find. having it in the middle makes it hard to see..
So the button order:
1. Add item
2. Add from master list
3. Request Suggested (?)
4. Print
5. More

Reply:
Ah yup!